handlers/list.ts

```diff
@@ -0,0 +1,136 @@
+interface IShoppingList {
+    onListUpdate(callback: (data: ListData[], ids: string[]) => void): void
+    onUsersUpdate(callback: (data: Member[], ids: string[]) => void): void
+    addToList(item: ListData): Boolean;
+    updateItemInList(item: ListData, id: string): Boolean;
+    deleteFromList(id: string): Boolean;
+    buyFromList(item: ListData, id?: string): Boolean;
+}
+
+class ShoppingListHandler implements IShoppingList {
+    readonly gun: Gun;
+    readonly user: UserGunDB;
+
+    constructor(gun: Gun) {
+        this.gun = gun;
+        user = gun.user();
+    }
+
+    public onListUpdate(callback: (data: ListData[], ids: string[]) => void): void {
+        let groupId = this.getUserGroupId()
```

Comment:
`groupId` bliver brugt i næsten alle funktion så  kan passende laves til en property

Reply:
Removed getUserGroupId function and set in constructor